### ISSUE TITLE
Not just showing indication but ability to cache route?

### ISSUE THREAD

**Issue by shirshak55** [wilburpowery/RoutesCached#3]
I guess it's just a matter of calling one artisan command? I think we can add this feature ?

What do you say?

**Comment by shirshak55** [comment]
Wow its is todo list also :)

**Comment by wilburpowery** [comment]
Yes. This is something I'm looking to do but can't find the time right now. If you have time, feel free to open a PR, I'll review it and merge it in.
